test/e2e/util.go

```diff
@@ -444,11 +444,14 @@ func DeleteRC(c *client.Client, ns, name string) error {
 	return nil
 }
 
-// Launch a Replication Controller and wait for all pods it spawns
-// to become running. The controller will need to be cleaned up external
-// to this method
+// RunRC Launches (and verifies correctness) of a Replication Controller
+// It will waits for all pods it spawns to become "Running".
+// The controller will need to be cleaned up external to this method.
 func RunRC(c *client.Client, name string, ns, image string, replicas int) error {
 	var last int
+
+	//TODO: We might want to make "maxContainerFailures" a factor of the replicas.
```

Comment:
I suggest just making it something in the range of .5% - 5% of replicas, minimum 1. That seems clearly better than a straight 10.